ros_ign_utils/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ros_ign_utils</name>
+  <version>0.0.0</version>
+  <description>Utility functions and applications for Gazebo ROS interactions</description>
```

Comment:
This sounds very similar to `ros_ign_gazebo` - what do you imagine the difference between them being?

Reply:
> inflate the dependencies of ros_ign_gazebo

Are you referring to `rclcpp_components`?

> without necessarily being something that every user would need/want.

Yeah it's good to not pay for what you don't use, but if one thing, it probably should be the other way around - the "utils" package has as few dependencies as possible.

Reply:
> ros_ign_gazebo was specifically for the tasks of starting the simulator as well as spawning.

I've been thinking of it more broadly as a package to hold simulation-specific functionality, to separate it from `ros_ign_bridge`, which is only concerned with bridging transport

Reply:
> Are you referring to rclcpp_components?

As well as geometry_msgs, sensor_msgs, tf2 and tf2_ros, all of which need to be added to package.xml :upside_down_face:

Reply:
> I've been thinking of it more broadly as a package to hold simulation-specific functionality, to separate it from ros_ign_bridge, which is only concerned with bridging transport

In that case, I will move it over, if it's ever an issue, it should be trivial to separate out as necessary.